starter_project_backend_nest/src/auth/auth.service.ts

```diff
@@ -14,8 +15,9 @@ export class AuthService {
 
   async validateUser(email: string, password: string): Promise<any> {
     const user = await this.userService.findOne(email);
```

Comment:
before checking the password it's better to add  catch block if the user doesn't exist